Resources/views/CRUD/list_url.html.twig

```diff
@@ -42,7 +42,14 @@ file that was distributed with this source code.
             {% set value = value|replace({'http://': '', 'https://': ''}) %}
         {% endif %}
 
-        <a href="{{ url_address }}">{{ value }}</a>
+        <a
+            href="{{ url_address }}"
+            {%- for attribute, value in field_description.options.attributes|default([]) %}
```

Comment:
It will generate wrong html code than, like this:

> com"target="_blank"class="fooLink">htt

So, it is not miss, it is for purpose.